subscription-service/src/main/resources/db/migration/VO_16_initialize_ geo_property.sql

```diff
@@ -0,0 +1,2 @@
+alter table geometry_query
```

Comment:
in the name of the file, you used a `O` instead of a `0` (and there is an extra space after `initialize_`). so it is not executed.

Reply:
before initializa no ?

Reply:
?

Reply:
the extra space is before initialize ? and no after ?

Reply:
the file is currently named `VO_16_initialize_ geo_property.sql`, I let you find the extra space in it.

Reply:
it's corrected